userapi/src/main/java/fk/prof/userapi/verticles/HttpVerticle.java

```diff
@@ -69,10 +71,12 @@ private Router configureRouter() {
     public void start(Future<Void> startFuture) throws Exception {
         aggregationWindowDurationInSecs = userapiConfigManager.getAggregationWindowDurationInSecs();
         baseDir = userapiConfigManager.getBaseDir();
+        httpConfig = config().mapTo(Configuration.HttpConfig.class);
```

Comment:
redundant? we can get this from configuration class directly right, rather than deserializing to httpconfig again